flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -307,6 +307,20 @@ public static String[] mergeListsToArray(List<String> base, List<String> append)
                     .defaultValue(10)
                     .withDescription("The maximum number of old log files to keep.");
 
+    /**
+     * This option is here only for documentation generation, it is only evaluated in the shell
+     * scripts.
+     */
+    @SuppressWarnings("unused")
+    public static final ConfigOption<Boolean> FLINK_STD_REDIRECT =
+            ConfigOptions.key("env.std.redirect")
```

Comment:
Maybe the key name could be `env.stdout-err.redirect-to-file`. It is more explicit.